pkg/node/devnode.go

```diff
@@ -72,6 +72,7 @@ type DevBee struct {
 	errorLogWriter   io.Writer
 	apiServer        *http.Server
 	debugAPIServer   *http.Server
+	quitChunk        chan bool
```

Comment:
`chan bool` is not required. You can instead use `chan struct{}`. This is more optimal.